orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -47,7 +51,17 @@ class CreateWebhookTask implements RetryableTask {
     def customHeaders = stage.context.customHeaders
     boolean waitForCompletion = (stage.context.waitForCompletion as String)?.toBoolean()
 
-    def response = webhookService.exchange(method, url, payload, customHeaders)
+    def response
+    try {
+       response = webhookService.exchange(method, url, payload, customHeaders)
+    } catch (HttpServerErrorException e) {
+      def statusCode = e.getStatusCode()
+      if (statusCode.is5xxServerError() || statusCode.value() == 429) {
```

Comment:
Are you sure this will be able to catch the 429? Isn't that a client exception? AFK, but I think you should catch the super class instead (HttpStatusCodeExveption)?